selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/inject/SeleniumTestHandler.java

```diff
@@ -170,18 +171,23 @@ public void onFinish(ITestContext context) {}
 
   @Override
   public void onStart(ISuite suite) {
-    runningTests.clear();
```

Comment:
I guess it was added for a good reason. 
I would like to keep it here unless it brings issue.

Reply:
I was added it before to solve the problem which had gone since that time and now it courses another problem: missing clean up step for some test which being executed in the **CheOneThreadTestsSuite.xml** suite bacause when **CheSuite.xml** suite in turn starts execution, this command ** runningTests.clear()* removes that test from **SeleniumTestHandler::runningTests** list and so prevents removal of test workspace which belongs to that test when SeleniumTestHandler::shutdown() method is executed;